vendor/github.com/onsi/gomega/matchers/succeed_matcher.go

```diff
@@ -34,7 +34,7 @@ func (matcher *SucceedMatcher) FailureMessage(actual interface{}) (message strin
 	if errors.As(actual.(error), &fgErr) {
 		return fgErr.FormattedGomegaError()
 	}
-	return fmt.Sprintf("Expected success, but got an error:\n%s\n%s", format.Object(actual, 1), format.IndentString(actual.(error).Error(), 1))
+	return fmt.Sprintf("Expected success, but got an error:\n%s", format.Object(actual, 1))
 }
 
 func (matcher *SucceedMatcher) NegatedFailureMessage(actual interface{}) (message string) {
```

Comment:
Based on the given code patch, here are a few suggestions:

1. Ensure that the `actual` parameter passed to `FailureMessage()` and `NegatedFailureMessage()` is of the expected type i.e., an error. Failing to do so may cause unexpected runtime errors.

2. Consider updating the error message returned from `FailureMessage()`, when an error is received, to include the actual error message instead of just its type.

3. Update the implementation of `NegatedFailureMessage()` to provide a meaningful message when the matcher is negated (i.e., Expect().NotTo(Succeed())).

4. Check if the import for `format` package is correct.

5. Use named return values in `FailureMessage()` and `NegatedFailureMessage()` functions to improve the readability of code.

I hope this helps!